repos.csv

```diff
@@ -2994,43 +2994,42 @@ scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildActi
 ,FasterXML/StaxMate,master,maven,,,,,,
 ,FasterXML/TransiStore,master,maven,,,,,,
 ,FasterXML/aalto-xml,master,maven,,,,,,
-,FasterXML/jackson-annotations,2.14,,,,,,,
-,FasterXML/jackson-benchmarks,2.13,,,,,,,
-,FasterXML/jackson-core,2.14,maven,,,,,,
-,FasterXML/jackson-databind,2.14,,,,,,,
+,FasterXML/jackson-annotations,2.17,,,,,,,
+,FasterXML/jackson-benchmarks,2.15,,,,,,,
+,FasterXML/jackson-core,2.17,maven,,,,,,
+,FasterXML/jackson-databind,2.17,,,,,,,
```

Comment:
Did notice that there's also for instance a https://github.com/FasterXML/jackson-databind/tree/master, which has the same commit as the 2.17 main branch. Not sure what their flow is specifically, but if they consistently also maintain `master` or `main` we might want to switch to that.

Reply:
I am also unsure. The 2.17 branch is their default branch in GitHub. But the next release is indeed being developed on master. I am unsure if the next release will be 3.0 or 2.18.